build.rs

```diff
@@ -157,11 +157,9 @@ fn write_testsuite_tests(
     writeln!(out, "#[test]")?;
     if x64_should_panic(testsuite, &testname, strategy) {
         writeln!(out, r#"#[should_panic]"#)?;
-    } else if ignore(testsuite, &testname, strategy) {
+    // Ignore when using QEMU for running tests (limited memory).
+    } else if ignore(testsuite, &testname, strategy) || pooling && platform_is_emulated() {
```

Comment:
Could this use `(` and `)` to group the latter condition here? I never remember the precedences of `||` and `&&`...